### PR TITLE
Portfolio project submission: My works and about sections

### PR DESCRIPTION
**Pull request by dvcorreia1984**
Completed the first submission of My portfolio project for feedback.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,47 +15,215 @@
...
+      </div>
+    </div>
+  </section>
 
-      <div class="page1bg"></div>
+  <section id="page2">
+    <div class="contentbox">
+      <header>
+        <h1 class="works">My Recent<br> Works</h1>
+        <div class="line"></div>
```
Comment (mahbubul14) [inline index.html]: - [ ] You have multiple h1 tag on your web page.  Kindly change this h1 to other hx(like h2,h3 etc.) tag. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browser. :+1:

File: index.html [hunk 2]
```diff
@@ -15,47 +15,215 @@
...
+</div>
+</div>
+</div>
 
-  <footer class="footer">
-  
-  </footer>
+<div class="about">
+  <div>
+    <h1 id="h1p3">About me</h1>
```
Comment (mahbubul14) [inline index.html]: - [ ] You have multiple h1 tag on your web page.  Kindly change this h1 to other hx(like h2,h3 etc.) tag. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browser. :+1:

File: index.html [hunk 3]
```diff
@@ -15,47 +15,215 @@
...
+                      <li><div class="button2"><button2>CSS</button2><div><li>  
+                      <li><div class="button2"><button2>Javascript</button2><div><li>
+                      <li><div class="button2"><button2>HTML</button2><div><li>    
+                    </ul>
+                    <div class="tilebut"><button>See Project</button><div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] As per the project requirements, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using the templates in Figma.  The font size on your webpage appears to be too small. Please make an effort to align it with the font size specified in the Figma design. Kindly do the change to the whole website and try to match the Figma design as much as possible. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign | figma

File: index.html [hunk 4]
```diff
@@ -15,47 +15,215 @@
...
+            <div class=whitespace>
+              <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
+                  <ul class="minimenu">
+                    <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
+                    <li><div class="button2"><button2>CSS</button2><div><li>  
+                    <li><div class="button2"><button2>Javascript</button2><div><li>
+                    <li><div class="button2"><button2>HTML</button2><div><li>    
+                  </ul>
+                  <div class="tilebut"><button>See Project</button><div>
+              </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] Kindly implement the button interactions (enable, hover, etc.) for all the buttons in the page and try to match the Figma design as well :+1: 

Here is the Figma design:

hover_figma

File: index.html [hunk 5]
```diff
@@ -15,47 +15,215 @@
...
+          <li class="button3">Version Control</li>   
+        </ul>
+
+        <ul class="minimenu"> 
+          <li class="button3">CLI</li>
+          <li class="button3">Web Developement</li>  
+          <li class="button3">API Design</li>  
+        </ul>
+  </div>
+</div>
```
Comment (mahbubul14) [inline index.html]: - [ ] The technologies should be in the middle of each category. Kindly do the change and try to match the Figma design as well. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign | figma_about

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
**Hi @dvcorreia1984,**

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Pull Request should have a descriptive title telling you what changes are inside and a short summary describing introduced changes in more detail. Kindly work on it.

pr

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Clean code.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (dvcorreia1984) [reply index.html]: Thank you for your detailed feedback Mahbub.

File: index.html [hunk 6]
```diff
@@ -15,47 +14,232 @@
...
+          <a href="#page2" class="action-button"><i class="fa fa-chevron-down" aria-hidden="true"></i></a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="page2">
+    <div class="contentbox">
+      <header>
+        <h1 class="works">My Recent<br> Works</h1>
```
Comment (victorokoroji) [inline index.html]: - [ ] As suggested by the previous reviewer,  You have multiple `<h1>` tags on your web page. Kindly change this `<h1>` to other `hx(like h2,h3 etc.)` tag. It is better for your page to have just one `<h1>` tag, this makes your website semantic and easily readable by browsers. 👍

File: index.html [hunk 7]
```diff
@@ -15,47 +14,232 @@
...
+      </ul>
+      <button class="button">See Project</button>
+    </div>
+  </div>
+</div>
+
+
+<div class="about">
+  <div>
+    <h1 id="h1p3">About me</h1>
```
Comment (victorokoroji) [inline index.html]: - [ ] As suggested by the previous reviewer,  Kindly change this `<h1>` tag to another `heading tag(like h2,h3 etc.)` tag. It is better for your page to have just one `<h1>` tag, this makes your website semantic and easily readable by browsers. 👍

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @dvcorreia1984, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️

- [ ] The font size on your webpage appears to be too small. Please make an effort to align it with the font size specified in the Figma design. Kindly do the change to the whole website and try to match the Figma design as much as possible. 👍

- [ ] As suggested by the previous reviewer,  You have multiple `<h1>` tags on your web page. Kindly change the `<h1>` to other `hx(like h2,h3 etc.)` tag apart from the one in the headline section. It is better for your page to have just one `<h1>` tag, this makes your website semantic and easily readable by browsers. 👍


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dvcorreia1984** [comment]
Thank you. I made these changes.

Regards. Victor.

Op Sa. 3 Jun. 2023 om 23:56 het Victor Ebube Okoroji <
***@***.***> geskryf:

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @dvcorreia1984, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**